kong/clustering/init.lua

```diff
@@ -5,10 +5,54 @@ local pl_file = require("pl.file")
 local pl_tablex = require("pl.tablex")
 local ssl = require("ngx.ssl")
 local openssl_x509 = require("resty.openssl.x509")
+local ngx_null = ngx.null
+local ngx_md5 = ngx.md5
+local tostring = tostring
+local assert = assert
+local concat = table.concat
+local sort = table.sort
+local type = type
 
 
 local MT = { __index = _M, }
 
+local function table_to_sorted_string(t)
```

Comment:
let's have some basic test case for this function, just to catch any potential
behaviour change in LuaJIT or penlight for example.